Azrux/Athenas-front
# Product Card

## Description
Create ProductCard component

## Screenshots
![image](https://github.com/Azrux/Athenas-front/assets/89431511/9d4f8aa6-9d8d-4eb0-8056-6c308300c593)
![image](https://github.com/Azrux/Athenas-front/assets/89431511/f6ca98b8-cd6e-415e-8849-8dab1ac75cba)

## How to test?
Run:` npm run dev`
Go to [http://localhost:3000/test](http://localhost:3000/test)
Check the card is working
Check the responsive design
